packages/visualizations/src/components/MapPoi/index.ts

```diff
@@ -0,0 +1,12 @@
+import type { FeatureCollection } from 'geojson';
+import type { PoiMapOptions } from './types';
+import PoiGeoJsonImpl from './PoiGeoJson.svelte';
+import SvelteImpl from '../SvelteImpl';
+
+import 'maplibre-gl/dist/maplibre-gl.css';
```

Comment:
There must be a reason why we did it in this file instead of the component, but I'm not sure. Maybe it was only to bundle the CSS only once for the two maplibre components, and we could have done it once in MapRender already. But if nobody remembers, we can do it the clean way as you say and see if something breaks 😄

Reply:
We can leave it like this for this PR in the spirit of _"do as close as we did for the other maps and refactor later"._